src/AppInstallerCLICore/Workflows/ResumeFlow.cpp

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+#include "pch.h"
+#include "ResumeFlow.h"
+
+namespace AppInstaller::CLI::Workflow
+{
+    void Checkpoint::operator()(Execution::Context& context) const
+    {
+        if (!Settings::ExperimentalFeature::IsEnabled(Settings::ExperimentalFeature::Feature::Resume))
+        {
+            return;
+        }
+
+        if (WI_IsFlagSet(context.GetFlags(), Execution::ContextFlag::Resume))
+        {
+            // TODO: If the resume flag is set, resume workflow behavior.
```

Comment:
Since the `operator<<` overloads will need to know when to skip tasks (assuming that is the path chosen), it seems better to let them also handle knowing when to stop skipping tasks.  Alternatively, a new workflow virtual could be added that allows for this skip/resume decision.